src/main/java/io/blamer/hub/pg/TokenValidated.java

```diff
@@ -0,0 +1,57 @@
+package io.blamer.hub.pg;
+
+import io.blamer.hub.exceptions.TokenAlreadyExists;
+import io.blamer.hub.model.Token;
+import io.blamer.hub.model.Tokens;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.r2dbc.core.DatabaseClient;
+import org.springframework.stereotype.Component;
+import reactor.core.publisher.Mono;
+@Component
+public class TokenValidated implements Tokens {
```

Comment:
@iColdLight It's better to name this class just like `Validated`, since your interface already saying that this is Tokens

Reply:
I didn't name my class Validated because we already have a class with that name